camunda-utils/Camunda/Camunda.py

```diff
@@ -63,6 +68,34 @@ def _map_secrets(self):
         # Set Robot variable
         built_in.set_global_variable(SECRET_VARIABLE, secrets_wrapper)
 
+    @keyword(name="Throw BPMN Error")
+    def throw_bpmn_error(self, errorCode: str, errorMessage: str):
```

Comment:
Does this allow `errorMessage` to be optional? The API for BPMN errors does, and so the Worker has followed suit